sonar/modules/documents/dojson/rerodoc/model.py

```diff
@@ -218,7 +226,7 @@ def marc21_to_provision_activity_field_260(self, key, value):
             })
 
         # End date
-        if len(years) > 1:
+        if len(years) > 1 and re.match(r'^[0-9]{4}$', years[1]):
```

Comment:
you can replace `[0-9]` by `\d`, see https://docs.python.org/3/library/re.html for more details